### PR TITLE
Improve documentation about manual database migration

### LINKED ISSUE #23246: Improve documentation about manual database migration
### Description

As it's stated in the docs for manual database migration:
`The required changes are written to an SQL file that you can review and manually run against the database.`

It's not very clear what SQL file is used and where it is located. It could be improved.

https://www.keycloak.org/docs/latest/upgrading/index.html#manual-relational-database-migration

### PR DESCRIPTION
**Pull request by mabartos**
Closes #23246

@vmuzikar @stianst Can you check it? Thanks

### CODE REVIEW

File: docs/documentation/upgrading/topics/migrate_db.adoc [hunk 1]
```diff
@@ -7,8 +7,8 @@ database is automatically migrated when you start the new installation for the f
 
 === Automatic relational database migration
 
-To enable automatic upgrading of the database schema, set the migration-strategy property value to "update" for the
-default connections-jpa provider:
+To enable automatic upgrading of the database schema, set the `migration-strategy` property value to "update" for the
+default `connections-jpa` provider:
```
Comment (vmuzikar) [inline docs/documentation/upgrading/topics/migrate_db.adoc:11]: I think `migration-strategy=update` is the default, no?

**Review by vmuzikar — COMMENTED** [review]
Thanks @mabartos!

I'd like to also ask for a review the @keycloak/store team.

File: docs/documentation/upgrading/topics/migrate_db.adoc (hunk 1 above)
Comment (mabartos) [reply docs/documentation/upgrading/topics/migrate_db.adoc:11]: Yep, that's right. Could also be clarified. Nice catch!

**Review by vmuzikar — APPROVED** [review]

**Review by mhajas — APPROVED** [review]
Thanks @mabartos!

**Pull request merged**